src/pages/team/index.tsx

```diff
@@ -263,15 +256,29 @@ const IconLink: FunctionComponent<{className?: string}> = ({
   </svg>
 )
 
-export const getServerSideProps: GetServerSideProps = async function (
+export const getServerSideProps: GetServerSideProps<TeamPageProps> = async function (
   context: any,
 ) {
   const {eggheadToken} = getTokenFromCookieHeaders(
     context.req.headers.cookie as string,
   )
 
   const {data: teams} = await loadTeams(eggheadToken)
-  const team = teams[0] || null
+
+  let team: TeamData | undefined
+
+  const fetchedTeam = teams[0]
+  if (fetchedTeam) {
+    team = {
+      inviteUrl: `${process.env.NEXT_PUBLIC_DEPLOYMENT_URL}/team-invite/${fetchedTeam.invite_token}`,
+      members: fetchedTeam.members,
+      numberOfMembers: fetchedTeam.number_of_members,
+      capacity: fetchedTeam.capacity,
+      isFull: fetchedTeam.is_full,
+      accountSlug: fetchedTeam.slug,
+      stripeCustomerId: fetchedTeam.stripe_customer_id,
+    }
```

Comment:
Normalize data right out of the API request instead of leaving that to the component.